pkg/controller/node/node_controller.go

```
@@ -136,13 +131,13 @@ func (r *ReconcileNode) Reconcile(request reconcile.Request) (reconcile.Result,
 func (r *ReconcileNode) findRelevantFileIntegrities(currentnode *corev1.Node) []*fiv1alpha1.FileIntegrity {
 	resultingFIs := []*fiv1alpha1.FileIntegrity{}
 	fiList := fiv1alpha1.FileIntegrityList{}
-	r.client.List(context.TODO(), &fiList)
+	_ = r.client.List(context.TODO(), &fiList)
```

It's actually not entirely clear to me why we're ignoring errors here :confused:

---

Yeah we should handle them

---

I thought ignoring was acceptable since we'd end up with an empty list and at worst do nothing, but yeah, handling the errors is safer.